README.md

```diff
@@ -384,22 +455,50 @@ The use case of "pairing" two DIDs by delegating all current and future resource
 The format for this scheme is as follows:
 
 ``` abnf
-ownershipscheme = "my:" kind
-kind = "*" / <scheme> 
+ownershipscheme = "my:" target
+target = "*" / <scheme> / <uri>
 ```
 
+### 4.2.1 `my` Delegation By Parenthood
+
+Capabilities being delegated by parenthood (the "apex" delegation) MUST be prefixed with `my:`. Including this prefix disambiguates that the source of this capability is the current issuer's owned resources.
```

Comment:
My current interpretation of this is that "parenthood" mean self issued either by a user itself or a service. An issuer is telling an audience, "i own this resource".
Can that audience redelegate using `my:`? No, that what `as:` is for right ? So is it correct to think that a delegation using `my:` can only be redelegated using `as:` ?
If yes, what happens if we want to use `ucan:` ?

Reply:
> My current interpretation of this is that "parenthood" mean self issued either by a user itself or a service. An issuer is telling an audience, "i own this resource".

Correct.

> Can that audience redelegate using `my:`? No

Correct.

> that what `as:` is for right ?

No, that's only the case for the `my:*` or `my:<scheme>` resource. These resources may be delegated further using `as:did:...`.

----

The whole idea behind this change was that now capability delegation chains begin with their resources being e.g. `my:wnfs:/matheus23.fission.name/` instead of just `wnfs:/matheus23.fission.name/`.
Then if you want to delegate that capability, you'll use `wnfs:/matheus23.fission.name/` as resource in newer UCANs.

Before, when we saw `wnfs:/matheus23.fission.name/` it was unclear whether that was meant to be introduced by parenthood or whether it was meant to be delegated, the `my:` prefix makes it unambiguous.

Reply:
>>   that what as: is for right ? So is it correct to think that a delegation using my: can only be redelegated using as: ?
>
> No, that's only the case for the my:* or my:<scheme> resource. These resources may be delegated further using as:did:....

@matheus23 how is this different from what @hugomrdias said ? I think you either both saying the same thing or am I'm missing something

Reply:
The whole `my:*` -> `as:did:key:zAlice` bothered and it just occurred to me, why not just stick to `as:` scheme e.g. `did:key:zAlice` could express `my:*` via `as:did:key:zAlice:*` and everything downstream will continue with a same or more refined `with`.

Validator will know whether `as:did:key:zAlice:*` is delegated or self issued (Delegation By Parenthood) based on whether DID of the issuer matches the DID in `as`.

Reply:
> You can't use ucan:* to (re-)delegate my: or as: capabilities. You should be able to do that with any other capability.

What the rational here ? As in why should re-delegation with `ucan:*` is not permitted ? Also what should happen if referenced proof only delegates `my:*` ?

Reply:
> @matheus23 how is this different from what @hugomrdias said ? I think you either both saying the same thing or am I'm missing something

@hugomrdias said "a delegation using my: can *only* be redelegated using as:", and I was trying to say "No, it can *also* be delegated using e.g. `wnfs:...`, when the original resource was `my:wnfs:...`".

I was thinking there exists some confusing around resources that look like `my:*` or `my:email`, and resources that look like `my:wnfs:/matheus23.fission.name/` or `my:email:matheus23@email.com`. The former can only be re-delegated using `as:` the latter can be re-delegated by dropping the `my:` prefix.

Reply:
> The whole `my:*` -> `as:did:key:zAlice` bothered and it just occurred to me, why not just stick to `as:` scheme e.g. `did:key:zAlice` could express `my:*` via `as:did:key:zAlice:*` and everything downstream will continue with a same or more refined `with`.
> 
> Validator will know whether `as:did:key:zAlice:*` is delegated or self issued (Delegation By Parenthood) based on whether DID of the issuer matches the DID in `as`.

Yes, that'd work and we've considered it. I don't exactly remember why we've chosen to go for `my:`. Probably just fewer bytes? @expede do you remember?

Reply:
ok im sorry but im completely lost with usage of `my:`, `as:` and `ucan:`, maybe we can find some sync time to unpack this topic? IMO this all section could use some love 


> I assume you're speaking of a setup where e.g. the UCAN contains a single capability with the resource ucan:* and its proof contains a single capability my:*?
In that case, the UCAN doesn't have any capabilities. I.e. it would be equally powerful as a UCAN with an empty attenuation and proof array.

are you saying that we can omit the att for full cap delegation ?

Reply:
> Probably just fewer bytes? @expede do you remember?

There's a few reasons:

* `my:` came first, and we added `as:` later to disambiguate a specific edge case
* `my:` requires fewer bytes

We can potentially only use `as:` for both since we can infer parenthood from this information. It is a bit annoying that we end up embedding the entire DID just to introduce something by parenthood, but worth investigating at least.

Reply:
@hugomrdias Apologies for the late replies, I was out most of last week with COVID. Happy to get on a sync call: https://calendly.com/expede/

> IMO this all section could use some love

Good feedback; thanks!

> im completely lost with usage of my:, as: and ucan:, maybe we can find some sync time to unpack this topic?

I'll take a shot at clarifying these here in case it's helpful:

### `ucan:`

`ucan:` is a way of tagging some data as being a UCAN; for example when redelegating everything from a proof (to save space), you can say "everything from this one"

### `my:`

`my:` has two roles.

#### Role 1: Explicit Parenthood

It is used to show that the issuer is introducing some capability from their ("ambient") owned resources — things they created and are the "root user" for. Being explicit avoids an edge case where an unrelated proof can hijack the provenance of something you own, preventing you from using it because it looks like it's being brought in from this bad source. This looks like `my:dns:example.com`, and is only required on the initial delegation.

#### Role 2: Wildcard Resource Selection

There are many use cases where you want to delegate access to resources by type, including ones that haven't been created yet. For example, from my laptop, I may want to give access to my desktop for the ability to manage all of my DNS records, even ones that I haven't registered yet. This looks like `my:dns`. Delegating "everything" looks like `my:*`.

### `as:`

`as:` is a selector for working with things that have been delegated via `my:`, specifically when you want to wildcard them. If you want to redelegate the wildcard resources from a single source, like saying "`did:key:zBob` redelegates everything from `did:key:zAlice`" with `as:did:key:zAlice:*` or more importantly, all of Alice's DNS records (or whatever reousrce) from the `my:` wildcard example above: `as:did:key:zAlice:dns`.

Reply:
Also probably worth noting that these conversations are exactly why this is a collaborative process! If we can't justify a feature that's in this draft, it might need to be reworked or removed.

We have very concrete use cases for this stuff at Fission, but it's also possible that it could be reframed to be easier to work with. I'm open to changes, basically 👍

Reply:
@expede I was wondering if you had a chance to give another thought to our `"my": []` top level key proposal we've discussed on the call as an alternative to `my:` schema. We have evaluate how that would affect our codebase and so far it seems like it would be a real improvement.

If that idea is approved I'd like to make changes to our libraries to adopt it before we deploy UCANs is production so we can avoid carrying the legacy code for interop.